### ISSUE TITLE
Release 2 Deployment

### ISSUE THREAD

**Issue by YFrancis10** [MeMantraa/MeMantra#425]
## User Story
Hosting the backend and db on a machine so that it is not only available locally. Also deploying the frontend to android devices  

## Acceptance Criteria
- [ ] Given an android device, i should be able to use the app and accomplish some features 

## Story Points
7

## Priority
- High 

## Risk
- High
